source/sass/components/publication-page.scss

```diff
@@ -53,6 +53,11 @@
   height: 18px;
   margin-right: 10px;
   background-image: url(../_images/glyphs/download-file.svg);
+  transition: filter 0.2s ease-in-out, -webkit-filter 0.2s ease-in-out;
```

Comment:
I added this line as without it, the svg icon and the text of the button seemed to be out of sync when changing color. feel free to ping me if you would like me to show you what I mean :)

Reply:
Good call. This is how we implemented [`.btn-secondary` the external link version](https://github.com/mozilla/foundation.mozilla.org/blob/master/source/sass/buttons.scss#L131-L142) as well.

Two things here:

- Nice that you included vendor prefix! Good news is you don't have to worry about it on this repo! We have `autoprefix` in place so vendor prefixes will auto get added after compilation! 
- Since `.download-button::before` share quite a bit of rules as what we have for the [`.btn-secondary` the external link version](https://github.com/mozilla/foundation.mozilla.org/blob/master/source/sass/buttons.scss#L131-L142), let's create a new mixin to refactor things a bit. Have a look at the existing examples in [mixins/_buttons.scss](https://github.com/mozilla/foundation.mozilla.org/blob/master/source/sass/mixins/_buttons.scss) and give it a try. You can always ping me if you have questions. 🤓